static/bidder-info/33across.yaml

```diff
@@ -1,4 +1,7 @@
 endpoint: "https://ssc.33across.com/api/v1/s2s"
+# This bidder does not operate globally. Please consider setting "disabled: true" in European datacenters.
+geoscope:
+  - !EEA
```

Comment:
While this syntax works for documentation purposes, it could potentially be a problem in the future if we ever want to parse this field. `!` has special meaning in YAML as it is used to declare a piece of data's data type.

I ran this through the YAML parsing library used in PBS-Go and it parses it without any error but this field is ignored. If it is defined in quotes (`"!EEA"`) it is parsed correctly but we might want to have a conversation about whether we should proceed with that strategy where quotes are required or come up with an alternate syntax to express logical negation.

Reply:
ok - updated to quote "!EEA"